webrtc/videoroom/assets/src/room_ui.ts

```diff
@@ -45,24 +45,41 @@ export function getRoomId(): string {
   return document.getElementById("room")!.dataset.roomId!;
 }
 
-function elementId(streamId: string, type: "video" | "audio" | "feed") {
+function elementId(
+  streamId: string,
+  type: "video" | "audio" | "placeholder" | "feed" | "mutedAudioIcon"
+) {
   return `${type}-${streamId}`;
 }
 
 export function addVideoElement(
   stream: MediaStream,
   label: string,
-  muted: boolean = false,
-  isLocalVideo: boolean = false
+  isLocalVideo: boolean = false,
+  mutedAudio: boolean = false,
+  turnedOffVideo: boolean = false,
+  showMutedAudioIcon: boolean = false
```

Comment:
Are there any situation when we don't want to show muted audio icon?

Reply:
User own tile has always muted audio, but we want to show muted audio icon there, only if somebody has truned off his microphone